hunts/src/tagFilter.js

```diff
@@ -0,0 +1,12 @@
+export function filterWithTagToggled(tagList, tag) {
```

Comment:
nit: i would change the `filter` prefix here, it reads like a verb but I think it was intended as a noun